mlrun/config.py

```diff
@@ -80,7 +80,6 @@
     "default_project": "default",  # default project name
     "default_archive": "",  # default remote archive URL (for build tar.gz)
     "mpijob_crd_version": "",  # mpijob crd version (e.g: "v1alpha1". must be in: mlrun.runtime.MPIJobCRDVersions)
-    "hub_url": "https://raw.githubusercontent.com/mlrun/functions/{tag}/{name}/function.yaml",
```

Comment:
Did you verify that nobody overrides this using an env variable in deployments? As we discussed - if deployment overrides this parameter and it doesn't exist in the config, then you'll get an exception on initialization, so need to make sure nobody does that.

Reply:
Sure, I checked it and indeed no one is using it.